### DISCUSSION TITLE
Wrong changelog date for 0.32.1

### DISCUSSION THREAD

**Discussion by DiegoBaldassarMilleuno** [kovidgoyal/kitty#7071] [General]
Didn't want to submit a bug report for this because it's so minor, but the changelog for version 0.32.1 is currently showing the date 2024-02-26 instead of 2024-01-26.

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/commit/ae1bf69a3d45a7cd59cbc27775657e8706b944d8
